### ISSUE TITLE
OpenCL backend tflite gpu delegate

### ISSUE THREAD

**Issue by rose-jinyang** [terryky/tflite_gles_app#15]
Hello
How are you?
Thanks for contributing to this project.
I am going to use the OpenCL backend tflite runtime on RK3399 Ubuntu 18.04 aarch64.
I found that u provided a script to build tflite gpu delegate so file for aarch64. (https://github.com/terryky/tflite_gles_app/blob/master/tools/scripts/tf2.4/build_libtflite_r2.4_aarch64.sh)
Do u think that the gpu delegate so file built by your script can use the OpenCL on RK3399 Mali T860?
Thanks

**Comment by terryky** [comment]
yes, I think it will works although I've never tested it on RK3399.

**Comment by rose-jinyang** [comment]
I got the built gpu delegate so file "libtensorflowlite_gpu_delegate.so"
BTW where can I get the built main tensorflowlite library file?
I could not find the file.

**Comment by terryky** [comment]
./tensorflow/bazel-bin/tensorflow/lite/libtensorflowlite.so
could you read [README.md](https://github.com/terryky/tflite_gles_app#222-copy-tensorflow-lite-libraries-to-target-jetson--raspi)?

**Comment by rose-jinyang** [comment]
thanks

**Comment by rose-jinyang** [comment]
Did u try to build/use the GPU enabled tflite runtime for Python?
if so, could u let me know about the steps/methods?
Thanks

**Comment by terryky** [comment]
Unfortunatelly, no.
I'm not familiar with tflite-python-API.

**Comment by aidlearning** [comment]
> Did u try to build/use the GPU enabled tflite runtime for Python?
> if so, could u let me know about the steps/methods?
> Thanks

you can refer aid learning . it make the GPU enabled tflite runtime for Python...
